src/Dropdown/Dropdown.tsx

```diff
@@ -104,11 +113,14 @@ const PlainDropdown: React.FC<IDropdownProps> = ({
   }, [])
 
   useLayoutEffect(handlePosition, [])
+  const width = dropdownRef.current
+    ? `${dropdownRef.current.clientWidth}px`
+    : undefined
 
   return (
     <div className={className} data-test='dropdown' ref={dropdownRef}>
       {button}
-      <Content
+      <Content useButtonWidth={useButtonWidth} width={width}
```

Comment:
The term button confused me here for a second, as it's not really a button (or it can be, but it can be anything really). Should we maybe it something else (e.g. `trigger`)?